tools/run_tests/artifacts/artifact_targets.py

```diff
@@ -485,12 +485,8 @@ def targets():
         PythonArtifact('windows', 'x64', 'Python38'),
         PythonArtifact('windows', 'x64', 'Python39'),
         PythonArtifact('windows', 'x64', 'Python310', presubmit=True),
-        RubyArtifact('linux', 'x86-mingw32', presubmit=True),
```

Comment:
If removing the ruby artifact build on linux, distribtest_targets.py (and possibly package_targets.py) also need to be updated.

With this change, the Ruby Distribtests Linux are failing
https://source.cloud.google.com/results/invocations/4ea9f8c2-f310-4a9d-8d3e-ec1d5dc0ef26/targets/github%2Fgrpc%2Fdistribtests/tests
(some of them already are failing on v1.46.x but this change seems to make it worse).

Reply:
Oh good catch. Removed distrib test targest